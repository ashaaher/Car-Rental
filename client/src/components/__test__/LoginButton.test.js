import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import LoginButton from '../LoginButton';

it('LoginButton Page matches snapshot', () => {
  const wrapper = shallow(<LoginButton />);
  expect(wrapper).toMatchSnapshot();
})