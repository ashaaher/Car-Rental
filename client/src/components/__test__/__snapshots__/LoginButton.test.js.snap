// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginButton Page matches snapshot 1`] = `ShallowWrapper {}`;
